contracts/src/v0.8/dev/interfaces/OCR2DRRegistryInterface.sol

```diff
@@ -0,0 +1,178 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.6;
+
+/**
+ * @title OCR2DR billing subscription registry interface.
+ */
+interface OCR2DRRegistryInterface {
```

Comment:
hmm this doesn't seem to really abstract the billing - is that not possible? Looks pinned to a very specific subscription/commitment structure. Ideally one could set a new billing registry which uses a completely different model

Reply:
I'll put some more thought into generalizing this.
That would probably just look like taking out the subscription function signatures from the interface huh?

Reply:
more than that since your RequestBilling and getCommitment both depend on subIDs

Reply:
I leaned down the other subscription stuff.. but yeah the subIDs in RequestBilling will need to stay.
I agree that it can just be left blank if switching to non-subs.